src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Properties/Resources.resx

```diff
@@ -396,4 +396,7 @@
   <data name="UpdatedResourceGroup" xml:space="preserve">
     <value>Updated resource group '{0}' in location '{1}'</value>
   </data>
+  <data name="OperationFailedWithTimeOut" xml:space="preserve">
+    <value>Operation failed because a request time out.</value>
```

Comment:
Sorry for the confusion... I was actually agreeing with you :)  I'm changing it to "timed out".